docs/source/en/guides/download.md

```diff
@@ -237,8 +237,8 @@ huggingface-cli download --help
 
 If you are running on a machine with high bandwidth, you can increase your download speed with [`hf_transfer`](https://github.com/huggingface/hf_transfer), a Rust-based library developed to speed up file transfers with the Hub. To enable it, install the package (`pip install hf_transfer`) and set `HF_HUB_ENABLE_HF_TRANSFER=1` as an environment variable.
 
-<Tip warning={true}>
+<Tip>
 
-`hf_transfer` is a power user tool! It is tested and production-ready, but it lacks user-friendly features like progress bars or advanced error handling. For more details, please take a look at this [section](https://huggingface.co/docs/huggingface_hub/hf_transfer).
+`hf_transfer` now support progress bars since version `0.1.4`. Consider upgrading (`pip install -U hf-transfer`) if you plan to enable faster downloads. For more details, please take a look at this [section](https://huggingface.co/docs/huggingface_hub/hf_transfer).
 
 </Tip>
```

Comment:
I would still brand it as "power user" and with a warning. Even though we now have progress bars, it is still not as user-friendly as pure Python code. For example a ctrl+c in the terminal will not interrupt a download performed with `hf_transfer`. And also, it doesn't cost us must to brand as "power user" to lower expectations :smile: